### PR TITLE
Liquidsoap support.

### PR DESCRIPTION
**Pull request by toots**
This should take care of it..

### CODE REVIEW

File: packages/liquidsoap/liquidsoap.1.2.0/opam [hunk 1]
```diff
@@ -17,7 +17,7 @@ depends: [
   "ocamlfind"
   "pcre"
   "camomile"
-  "dtools"
+  "dtools" {< "1.0.0"}
```
Comment (yoriyuki) [inline packages/liquidsoap/liquidsoap.1.2.0/opam:20]: This looks suspicious.  Is it okay?

**Review by yoriyuki — COMMENTED** [review]
Sorry, may  be too late but I'm not sure the code I indicate.

**Comment by yoriyuki** [comment]
Yes, only liquidsoap.1.2.0 fails to build.

**Comment by toots** [comment]
Yeah that's a typo. Will push another PR shortly.

2018-04-06 4:34 GMT-05:00 Yoriyuki Yamagata <notifications@github.com>:

**Pull request merged**